python/servo/package_commands.py

```diff
@@ -0,0 +1,97 @@
+
+@CommandProvider
+class PackageCommands(CommandBase):
+    @Command('package',
+             description='Package Servo (currently, Android APK only)',
+             category='package')
+    @CommandArgument('--release', '-r', action='store_true',
+                     help='Package the release build')
+    @CommandArgument('--dev', '-d', action='store_true',
+                     help='Package the dev build')
+    @CommandArgument('--android',
+                     default=None,
+                     action='store_true',
+                     help='Package Android')
+    def package(self, release=False, dev=False, android=None, debug=False, debugger=None):
+        env = self.build_env()
+        if android is None:
+            android = self.config["build"]["android"]
+        binary_path = self.get_binary_path(release, dev, android=android)
+        if android:
+            if dev:
+                env["NDK_DEBUG"] = "1"
+                env["ANT_FLAVOR"] = "debug"
+                dev_flag = "-d"
+            else:
+                env["ANT_FLAVOR"] = "release"
+                dev_flag = ""
+
+            target_dir = os.path.dirname(binary_path)
+            output_apk = "{}.apk".format(binary_path)
+            try:
+                with cd(path.join("support", "android", "build-apk")):
+                    subprocess.check_call(["cargo", "run", "--", dev_flag, "-o", output_apk, "-t", target_dir,
+                                           "-r", self.get_top_dir()], env=env)
+            except subprocess.CalledProcessError as e:
+                print("Packaging Android exited with return value %d" % e.returncode)
+                return e.returncode
+        else:
+            print(binary_path)
+
+    @Command('install',
+             description='Install Servo (currently, Android only)',
+             category='package')
+    @CommandArgument('--release', '-r', action='store_true',
+                     help='Package the release build')
+    @CommandArgument('--dev', '-d', action='store_true',
+                     help='Package the dev build')
+    def install(self, release=False, dev=False):
+        try:
+            binary_path = self.get_binary_path(release, dev, android=True)
+        except BuildNotFound:
+            print("Servo build not found. Building servo...")
+            result = Registrar.dispatch(
+                "build", context=self.context, release=release, dev=dev
+            )
+            if result:
+                return result
+            try:
+                binary_path = self.get_binary_path(release, dev, android=True)
+            except BuildNotFound:
+                print("Rebuilding Servo did not solve the missing build problem.")
+                return 1
+
+        apk_path = binary_path + ".apk"
+        if not path.exists(apk_path):
+            result = Registrar.dispatch("package", context=self.context, release=release, dev=dev)
+            if result is not 0:
```

Comment:
The `is` binary operator is used to check references. i.e., (IIRC) it's possible for to instances of `0` to exist in memory, so this is not guaranteed to always pass. Whereas singletons, like `None`, `True`, and `False` only exist once in memory so `is` is okay. So this should maybe be an `!=` operator instead